### ISSUE TITLE
Review notes

### ISSUE THREAD

**Issue by anba** [tc39/intl-displaynames-v2#34]
1. IsValidDateTimeFieldCode ( field )
- Typo: "is called with arguments field" &rarr; "is called with argument field". Singular "argument" instead of "arguments".
- Explicitly define which column to use, i.e. "If field is listed in Table 1"  &rarr; "If field is listed in the “Code” column of Table 1". For example compare with <https://tc39.es/ecma262/#sec-quotejsonstring>, step 2.a.

2. Table 1: Codes For Date Time Field of DisplayNames
- Consistently use either "e.g.," or "e.g.". So "e.g." with or without trailing comma.
    - Note 1: ECMA-262 and ECMA-402 use both forms.
    - Note 2: It seems like British spelling rules normally prefer to omit the comma, whereas American spelling rules prefer to use a comma.
    - Note 3: Resources like <https://english.stackexchange.com/questions/16172/should-i-always-use-a-comma-after-e-g-or-i-e/351762#351762> mention that OUP states that no comma should follow "e.g.", so I guess we should follow that rule to adhere with the ISO/ECMA preference to use OED.
- "The field indicating the hour in day."
  - I'm not sure "in day" makes sense here, because "hour" isn't normally used in other contexts.
  - For example compare to "day", which can be either "day in year", "day in month", or "day in week".
- Similar issue for "minute" and "second" entries.
- Typo: "timezone name" -> "time zone name", so space between "time" and "zone".

4. Intl.DisplayNames ( locales, options )
- Normal convention should still be to always validate all options, even when unused.
- That means the `GetOption(options, "languageDisplay", ...)` step should happen before `If type is "language", then`. (But don't move assigning `displayNames.[[LanguageDisplay]]` outside of the if-step.)
    - For example compare to `SetNumberFormatUnitOptions()`.

5. Internal slots
- Second bullet point, no comma in an enumeration of two values:
  > "dialect", and "standard".
- `The display name language display fields under display name type "language" [...]` is kind of hard to parse, I think it's okay to simplify this a bit, because the reader should be able to infer that "dialect" and "standard" refers to the two previously mentioned fields:
> The value of fields "dialect" and "standard" must be a Record which must have fields with the names of one of the valid display name styles: "narrow", "short", and "long".
- "[...] with keys corresponding to a String value with the "type" given in Unicode Technical Standard 35 for the calendar used for formatting."
  - Instead of putting "type" in quotes, use backticks to output fixed-width, code style text, i.e. `type`.

6. Properties of Intl.DisplayNames Instances
- Fifth bullet point, no comma in an enumeration of two values:
  > "dialect", or "standard"
- "while and only while" -> "if and only if". ("while" implies a temporal condition, whereas here a logical condition should be used.)
- Maybe the whole bullet point should just be reworded to:
> [[LanguageDisplay]] is one of the String values "dialect" or "standard", identifying the language display kind. It is only used when [[Type]] has the value "language".

**Comment by FrankYFTang** [comment]
Dear @anba  Thank you so much for your careful review. I believe I addressed all your concerns in #35. Please take a look and file additional issue if you spot more. Thanks. Really appreciate your hard working on this.
